### PR TITLE
Update getting-started.mdoc

### PR DESCRIPTION
**Pull request by RedGhost2100**
Minor change for clarity and accuracy

Closes #

## Changes proposed:
- edited the first section of the quick walkthrough to slightly more clearly describe the order in which necessary items are obtained 
- edited 2nd and 3rd steps of section on setting up colonies so that it now accurately states which tool is used in placing the builder’s hut/town hall

### CODE REVIEW

File: src/content/wiki/tutorials/getting-started.mdoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ You should also gather as many resources as you can-wood, as well as cobble, coa
 
 ### Step 1: Supply Camp/Supply Ship
   
-Once you have chosen a location for your colony, you must place a {% item name="build_tool" /%}, the remaining camp/ship is just decoration. You can leave it or tear it down.
+Once you have chosen a location for your colony, you must place a {% item name="supply_camp_and_ship" /%}. Go inside the ship/camp and aquire the {% item name="build_tool" /%} . The remaining camp/ship is just decoration. You can leave it or tear it down.
```
Comment (Thodor12) [inline src/content/wiki/tutorials/getting-started.mdoc:37]: - "acquire"
- Extra space after build tool
- Not all styles continue to have a build tool/town hall inside of the supply camp as they can be crafted separately. So maybe state "and acquire the build tool if the supply camp contains one, else craft it"

File: src/content/wiki/tutorials/getting-started.mdoc [hunk 2]
```diff
@@ -123,13 +123,13 @@ Get enough materials and craft your Supply Ship/Camp, then place it by right-cli
...
 
 You can now right-click the {% building name="townhall" /%} block again to look what else you can configure in your colony, like your colony name!
 
 ### Step 3
 
-Before any other building can be built, you must craft the {% building name="builder" /%} block and use your {% item name="scan_tool" /%} to place it. Once you commit to the placement of the Builder's Hut (the green checkmark), the block will be placed and a Builder will be automatically assigned (or you can manually assign one with the best [traits](/wiki/systems/worker) for a Builder if you changed this in the setting tab in the {% building name="townhall" /%} its GUI).
+Before any other building can be built, you must craft the {% building name="builder's hut" /%} block and use your {% item name="build_tool" /%} to place it. Once you commit to the placement of the Builder's Hut (the green checkmark), the block will be placed and a Builder will be automatically assigned (or you can manually assign one with the best [traits](/wiki/systems/worker) for a Builder if you changed this in the setting tab in the {% building name="townhall" /%} its GUI).
```
Comment (Thodor12) [inline src/content/wiki/tutorials/getting-started.mdoc:132]: `{% building %}` tags rely on building ids, these are not display names, so you cannot say `{% building name="builder's hut" /%}` as this needs the building id (builder)

**Review by Thodor12 — CHANGES_REQUESTED** [review]
Hello, thank you for your contribution. Overall the texts looked fine but I've got a few remarks.

File: src/content/wiki/tutorials/getting-started.mdoc (hunk 1 above)
Comment (MotionlessTrain) [reply src/content/wiki/tutorials/getting-started.mdoc:37]: > Not all styles continue to have a build tool/town hall inside of the supply camp as they can be crafted separately. So maybe state "and acquire the build tool if the supply camp contains one, else craft it"

Afaik having a town hall and build tool in the supply camp and ship is a requirement of those, so this should be true for all of them

**Pull request merged**